packages/views/src/components/theme-provider/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react'
+
+import { useThemeStore } from './hooks/use-theme-store'
+import { ColorType, ContrastType, FullTheme, ModeType } from './types'
+
+export function ThemeProvider({ children }: { children: React.ReactNode }) {
```

Comment:
repeated from `ui/views/theme-selector`? 
i think we can delete all changes from the `views` package. use `ui` package as the new source of truth

Reply:
I actually raised another PR for changes in the new architecture which should address most of your comments:
https://github.com/harness/canary/pull/483

I may need the views and canary changes here for tomorrow's demo.